### PR TITLE
move the example configuration into "example" directory

### LINKED ISSUE #139: Make example config more useful to launch and follow
In the current code base the example config is spread over multiple files. Consolidate all the files into a single directory. This example configuration should be a template to create a proper working configuration

### PR DESCRIPTION
**Pull request by GiriDandu**
- Fixes #139

### CODE REVIEW

File: src/asciidoc/running.ad [hunk 1]
```diff
@@ -11,13 +11,13 @@ To test the example application on localhost run the following commands.
 * To setup the h2 database run (stored in current directory as deploydb.example.db).
 
 ----
-  java -jar build/libs/deploydb-0.1.0-all.jar db migrate deploydb.example.yml
+  java -jar build/libs/deploydb-0.2.1-all.jar db migrate example/deploydb.launch.yml
```
Comment (polynomial) [inline src/asciidoc/running.ad]: lets change this to 'deploydb-*-all.jar'

File: src/asciidoc/running.ad [hunk 2]
```diff
@@ -11,13 +11,13 @@ To test the example application on localhost run the following commands.
...
 ----
 
-* To run the server (it reads configuration from config/ directory).
+* To run the server (it reads the model configuration from example/launch-config/ directory).
 
 ----
-  java -jar build/libs/deploydb-0.1.0-all.jar server example.yml
+  java -jar build/libs/deploydb-0.2.1-all.jar server example/deploydb.launch.yml
```
Comment (polynomial) [inline src/asciidoc/running.ad]: same version thing here

File: src/asciidoc/running.ad [hunk 3]
```diff
@@ -11,13 +11,13 @@ To test the example application on localhost run the following commands.
 * To setup the h2 database run (stored in current directory as deploydb.example.db).
 
 ----
-  java -jar build/libs/deploydb-0.1.0-all.jar db migrate deploydb.example.yml
+  java -jar build/libs/deploydb-0.2.1-all.jar db migrate example/deploydb.launch.yml
 ----
 
-* To run the server (it reads configuration from config/ directory).
+* To run the server (it reads the model configuration from example/launch-config/ directory).
```
Comment (polynomial) [inline src/asciidoc/running.ad]: s/model/example/

File: src/asciidoc/running.ad (hunk 1 above)
Comment (GiriDandu) [inline src/asciidoc/running.ad]: Sure

File: src/asciidoc/running.ad (hunk 3 above)
Comment (GiriDandu) [inline src/asciidoc/running.ad]: not sure what this means?

Comment (polynomial) [inline src/asciidoc/running.ad]: i think it should say 'the example configuration'

Comment (GiriDandu) [inline src/asciidoc/running.ad]: ok, its "sed". i get it now

File: example/deploydb.launch.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ database:
 # DeployDB base configuration directory (cannonical path)
 # YAML configuration files for models - services, environments, pipelines,
 # webhooks and promotions - should be stored in respective sub-directories
-configDirectory: config
+configDirectory: example/launch-config
```
Comment (maheshkelkar) [inline example/deploydb.launch.yml]: This should be "model-config". 

Launch config is provided via deploydb.launch.yml

File: features/api/config.feature [hunk 5]
```diff
@@ -6,7 +6,7 @@ Feature: DeployDB config APIs
 
   Scenario: When the system is idle, reloading the shall succeed
 
-    Given DeployDb configuration directory path is "./config"
+    Given DeployDb configuration directory path is "./example/launch-config"
```
Comment (maheshkelkar) [inline features/api/config.feature]: should be: ./example/models-config

**Pull request merged**